doc/manual/tuto.md

```diff
@@ -0,0 +1,112 @@
+---
+slug: tutorial
+---
+
+# Managing users
+
+In this tutorial, you will learn how use Nickel to manage a list of users,
+and then export it as a YAML file for a program consuming it.
+
+## Step 1: Install nickel
+
+## Step 2: Think about the schema
+
+Our current YAML has different fields, such as `name`, `ssh-keys`,
+`is-admin` or `extra-groups`. To create a Nickel contract that produce
+what we want, and validates input, we need to think about the type of
+each field.
+
+Nickel provides a lot of types, but also structures. In our case,
```

Comment:
> Nickel provides a lot of types, but also structures.

I'm not entirely sure what you mean by "structures" here. I think this sentence could possibly be cut.

Reply:
Yeah, I see what you mean. I think that since the "structure" terminology isn't one that's widely used or well-defined it could potentially create additional confusion, but ultimately it's up to you how you want to phrase this. :smile: